internal/config/profile.go

```diff
@@ -65,6 +65,9 @@ const (
 	ContainerizedHostNameEnv = "MONGODB_ATLAS_IS_CONTAINERIZED"
 	GitHubActionsHostNameEnv = "GITHUB_ACTIONS"
 	AtlasActionHostNameEnv   = "ATLAS_GITHUB_ACTION"
+	MongoDBUniversityUserEnv = "MONGODB_UNIVERSITY_USER" // MongoDBUniversityUserEnv is used to separate MongoDB University users from default users
```

Comment:
who sets this var?

Reply:
are we inline with university folks about this?

Reply:
> who sets this var?

The MongoDB university team will set this var when running the CLI in their environment

> are we inline with university folks about this?

The Jira ticket doesn't provide context to answer this question. I was planning to reach out once the PR is merged so that they are aware that this is coming and to set the env var.

Reply:
I tagged the university team in the atlascli slack channel for awareness.

Reply:
Thanks @andreaangiolillo and @fmenezes - delighted to see this change and happy to ensure it's rolled out to all of labs.